lldb/test/API/macosx/riscv32-corefile/TestRV32MachOCorefile.py

```diff
@@ -0,0 +1,81 @@
+"""Test that all of the GPR registers are read correctly from a riscv32 corefile."""
+
+import os
+import re
+import subprocess
+
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class TestRV32MachOCorefile(TestBase):
+    NO_DEBUG_INFO_TESTCASE = True
+
+    def test_riscv32_gpr_corefile_registers(self):
```

Comment:
```suggestion
    @skipUnlessDarwin
    def test_riscv32_gpr_corefile_registers(self):
```

Reply:
I didn't think there was anything very Darwin specific here, figured ObjectFileMachO was built on Linux etc but PR linux testing says I'm wrong.

Reply:
It's the `mach` headers you're using I think.

Reply:
```
#include <mach-o/loader.h>
#include <mach/thread_status.h>
```

Reply:
whoops, you're right loader.h I def needed in this one where I'm creating a mach-o binary, without duplicating a lot.  not worth it.